File: Python/ArctanSeries.py
#!/usr/bin/env python
# Maclaurin Series for Arctan
# arctan(x) = x - x ** 3 / 3 + x ** 5 / 5 ...
def main():
	x = input("Enter x: ")
	k = input("Enter k: ")
	answer = x; i = 1; j = 1
	while(i < k):
		if (i == 1):
			answer -= (x ** (j + 2)) / (j + 2)
		elif (i % 2 == 0):
			answer += (x ** (j + 2)) / (j + 2)
		elif (i % 2 != 0):
			answer -= (x ** (j + 2)) / (j + 2)
		i += 1; j += 2
	print "Answer:", answer
main()

File: C++/ArctanSeries.cpp
// Maclaurin series for arctan(x)
#include <iostream>
#include <string>
#include <math.h>

double input(std::string message){
	double x = 0;
	std::cout << message;
	std::cin >> x;
	return x;
}

int main(){
	double x = input("Enter x: ");
	double k = input("Enter k: ");
	double answer = x;
	for (int i = 1, j = 1; i < k; i++, j += 2){
		if (i == 1){
			answer -= pow(x, (j + 2)) / (j + 2);
		}else if (i % 2 == 0){
			answer += pow(x, (j + 2)) / (j + 2);
		}else if (i % 2 != 0){
			answer -= pow(x, (j + 2)) / (j + 2);
		}
	}
	std::cout << "Answer: " << answer << std::endl;
}
